### PR TITLE
DCWL-9: [RP] Fix for country dropdown box issue.

### PR DESCRIPTION
**Pull request by rambabu-posa**

### CODE REVIEW

File: public/autocomplete/location-autocomplete.min.js [hunk 1]
```diff
@@ -1 +1,2 @@
...
...[+-])=|)("+je+")([a-z%]*)$","i"),Ne=["Top","Right","Bottom","Left"],De=function(e,t){return"none"===(e=t||e).style.display||""===e.style.display&&oe.contains(e.ownerDocument,e)&&"none"===oe.css(e,"display")},Le=function(e,t,n,r){var i,o,a={};for(o in t)a[o]=e.style[o],e.style[o]=t[o];for(o in i=n.apply(e,r||[]),t)e.style[o]=a[o];return i},qe={};oe.fn.extend({show:function(){return y(this,!0)},hide:function(){return y(this)},toggle:function(e){return"boolean"==typeof e?e?this.show():this.hide():this.each(function(){De(this)?oe(this).show():oe(this).hide()})}});var Pe=/^(?:checkbox|radio)$/i,Me=/<([a-z][^\/\0>\x20\t\r\n\f]+)/i,Fe=/^$|\/(?:java|ecma)script/i,Re={option:[1,"<select multiple='multiple'>","</select>"],thead:[1,"<table>","</table>"],col:[2,"<table><colgroup>","</colgroup></table>"],tr:[2,"<table><tbody>","</tbody></table>"],td:[3,"<table><tbody><tr>","</tr></tbody></table>"],_default:[0,"",""]};Re.optgroup=Re.option,Re.tbody=Re.tfoot=Re.colgroup=Re.caption=Re.thead,Re.th=Re.td;var Ie=/<|&#?\w+;/;!function(){var e=K.createDocumentFragment(),t=e.appendChild(K.createElement("div")),n=K.createElement("input");n.setAttribute("type","radio"),n.setAttribute("checked","checked"),
```
Comment (calinursaciuc) [inline public/autocomplete/location-autocomplete.min.js:2]: Is this a comment. Maybe need to be removed? Or is a statement?

Comment (rambabu-posa) [reply public/autocomplete/location-autocomplete.min.js:2]: Calin, I just copied from this repo "govuk-country-and-territory-autocomplete"
https://github.com/alphagov/govuk-country-and-territory-autocomplete/blob/master/dist/location-autocomplete.min.js

Please check it and suggest me on this

Comment (calinursaciuc) [reply public/autocomplete/location-autocomplete.min.js:2]: Is okay then

**Review by calinursaciuc — APPROVED** [review]

**Comment by calinursaciuc** [comment]
To be merged only after new scenario is added in acceptance test

**Pull request merged**